server/server.js

```diff
@@ -15,6 +16,9 @@ module.exports = () => {
   const app = express()
   const pubFolder = path.join(__dirname, 'public')
 
+  Raven.config(process.env.SENTRY_DSN).install()
```

Comment:
Can we wrap this in an `if (process.env.SENTRY_DSN)`? That way local development won't bork

Reply:
Awe. Can't believe I missed this considering we're literally doing the same thing right below it for `FORCE_HTTPS`. Thanks for the review! 👍